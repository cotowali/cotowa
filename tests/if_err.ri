if 1 + 1 {
} else if 1 + 2 {
} else {
}
if 1 == 1 {
} else if 1 > 0 {
}
