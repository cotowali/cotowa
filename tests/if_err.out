if_err.ri 1,4: non-bool type `int` used as if condition
if_err.ri 2,12: non-bool type `int` used as if condition
